naming/src/main/java/org/jboss/as/naming/deployment/ContextNames.java

```diff
@@ -307,7 +307,7 @@ public ManagedReference getReference() {
                                 if(!optional) {
                                     throw NamingLogger.ROOT_LOGGER.resourceLookupForInjectionFailed(getAbsoluteJndiName(), e);
                                 } else {
-                                    NamingLogger.ROOT_LOGGER.tracef(e,"failed to lookup %s", getAbsoluteJndiName());
+                                    NamingLogger.ROOT_LOGGER.infof(e,"failed to lookup %s", getAbsoluteJndiName());
```

Comment:
If changing this line, run a formatter while at it (missing space between parameters).

And ofc, needs i18n...